pkg/cmd/server/api/v1/types.go

```diff
@@ -377,6 +377,9 @@ type AssetConfig struct {
 	// MetricsPublicURL is the public endpoint for metrics (optional)
 	MetricsPublicURL string `json:"metricsPublicURL"`
 
+	// CLIDownloadPublicURLs holds links to download client tools, where key is the platform name and value is the download URL (optional)
```

Comment:
I think this is the only thing left in this PR, waiting for @jwforres confirmation about where we want this config.